guacamole-docker/bin/start.sh

```diff
@@ -321,6 +321,10 @@ END
         "ldap-user-search-filter" \
         "$LDAP_USER_SEARCH_FILTER"
 
+    set_optional_property         \
+        "ldap-follow-referrals" \
```

Comment:
Very minor nitpick, but please line up the trailing back-slash, for consistency with the rest of the code...